docs/09_firebase_deployment.md

```diff
@@ -0,0 +1,201 @@
+# Metrics firebase deployment
+
+> Summary of the proposed change
+
+Describe the process of deployment of the Metrics application to the Firebase Hosting and deploying the Cloud functions needed to create test data for the application.
+
+
+# References
+
+> Link to supporting documentation, GitHub tickets, etc.
+
+- [Getting started with Firestore](https://cloud.google.com/firestore/docs/client/get-firebase)
+- [Deploying to Firebase Hosting](https://firebaseopensource.com/projects/firebase/emberfire/docs/guide/deploying-to-firebase-hosting.md/)
+- [Getting started with Firebase Cloud Functions](https://firebase.google.com/docs/functions/get-started)
+
+# Motivation
+
+> What problem is this project solving?
+
+The Metrics application instance with test data accessible via a link.
+
+# Goals
+
+> Identify success metrics and measurable goals.
+
+- Web application is deployed to Firebase Hosting and available for everyone via the link.
+- The Cloud Functions required for creating test data are deployed.
+
+# Non-Goals
+
+> Identify what's not in scope.
+
+Deployment to other platforms is out of scope.
+
+# Design
+
+> Explain and diagram the technical design
+
+`Metrics Web Source` -> `Build` -> `Deploy` ->  `Firebase Hosting`
+
+`Metrics Web Cloud Functions` -> `Deploy` -> `Firebase Cloud Functions`
+
+> Identify risks and edge cases
+
+# API
+
+> What will the proposed API look like?
+
+
+## Before you begin
+
+Before you start, you should have the following installed:
+
+1. [Flutter](https://flutter.dev/docs/get-started/install) v1.15.3.
+2. [npm](https://www.npmjs.com/get-npm).
+
+## Creating a new Firebase project.
+
+1. Go to [Firebase Console](https://console.firebase.google.com/), login if needed, and tap the `Add project` button.
+2. Enter a project name and tap the `Continue` to proceed.
+3. On the next page, select whether you want to use Firebase Analytics or not and click the `Continue`/`Create project` to proceed.
+4. Select a Firebase Analytics account if you are using one and tap the `Create project`.  
+
+After a couple of seconds, your project will be ready, and you'll be redirected to the project console.  
+Your next step will be to create a Firestore database: 
+
+1. Go to the `Database` tab on the left bar and click the `Create database` button under the `Cloud Firestore`. 
+2. Select `Start in test mode` to create a database with no security rules for now and tap `Next`.
+3. Select your database location and click `Done`.
+
+After a while, your database will be created, and you will see the database console page.
+
+## Firebase configuration
+
+When your Firestore database is up, you need to add a Web application to your Firebase project,
+to be able to connect your web application with the Firestore database: 
+
+1. In the [Firebase Console](https://console.firebase.google.com/), open your project and go to
+ the project setting page (tap on the setting gear icon near the `Project Overview` on top of the 
```

Comment:
```suggestion
 the project settings page (tap on the setting gear icon near the `Project Overview` on top of the 
```

Reply:
maybe we just remove this part: "go to the project setting page" and do tapping right away?